plugins/Satellites/src/gui/SatellitesDialog.cpp

```diff
@@ -1624,6 +1624,7 @@ void SatellitesDialog::savePredictedIridiumFlares()
 	}
 	else
 	{
+		#ifdef ENABLE_XLSX
```

Comment:
The `*.xlsx` filter should be removed in this case, otherwise we'll have "do nothing" behavior while the user expects something useful.

Reply:
In principle yes. But this is actually disabled code. It can be reactivated if Iridium-like satellites will be ever launched again.

Reply:
If we are adding this `#ifdef`, we must fix the surrounding code. Otherwise, if it's ever enabled, it will work in ways unexpected by the user.

Reply:
I've added some cosmetics for this disabled code, but in the future we should enable this code for flared satellites (after modification to support other flared sats)